projects/kit/utils/mask/create-time-mask.ts

```diff
@@ -1,13 +1,39 @@
 import {TuiTimeMode} from '@taiga-ui/cdk';
 import {TUI_DIGIT_REGEXP, TuiTextMaskList} from '@taiga-ui/core';
-import {TUI_TIME_MASK} from '@taiga-ui/kit/constants';
+import {MAX_TIME_VALUES} from '@taiga-ui/kit/constants';
+import {TuiTimeFormatParts} from '@taiga-ui/kit/types';
 
-export function tuiCreateTimeMask(mode: TuiTimeMode): TuiTextMaskList {
+function tuiCreateTimePartMask(
+    maxPartValue: number | undefined,
+    prefix?: string,
+): Array<string | RegExp> {
+    const partLength =
+        typeof maxPartValue === 'number' ? maxPartValue.toString().length : 0;
+
+    if (!partLength) {
+        return [];
+    }
+
+    const regExp = Array.from<undefined, RegExp | string>(
```

Comment:
`Array(length).fill(TUI_DIGIT_REGEXP)`

Reply:
Oh, just yesterday I recommended using a different format
https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/no-new-array.md

@mullinsmikey anyway, sorry about

Reply:
I'm skeptical about most of such rules :D